go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	k8s.io/client-go v0.18.2
 	k8s.io/gengo v0.0.0-20191120174120-e74f70b9b27e // indirect
 	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
```

Comment:
I don't see this used in the PR? Bad commit?

Reply:
I think this was left out from https://github.com/projectcontour/contour/pull/2727 - I noticed this yesterday. I guess we don't have any CI checks to ensure go.mod is up-to-date.

Reply:
I'm surprised it didn't yell on the build since it's marked as readonly. In any case then this is fine since it's missing.

Reply:
Yeh, `make check` yelled at me about it :(

Reply:
The same change is in #2746, so I'll merge that and rebase this on top.